set_address/models/sale_order.py

```diff
@@ -0,0 +1,30 @@
+from odoo import api, models
+
+
+class SaleOrder(models.Model):
+    _inherit = ["sale.order"]
+
+    @api.onchange("partner_id")
+    def onchange_partner_id(self):
+        """Overided this function so that if partner_id is company than check for
+        employees which has address type : delivery and set delivery address True.if we
+        find more than one employee first employee will be shown if condition is not
+        fullfilled then check for employees who has address type : dropship and if we
+        get more than one then the first employee will be shown if no condition is
+        fullfilled and than the employees who have not selected address type : delivery
+        than partner_shipping_id will be the company address #T00460"""
+        company = self.env["res.partner"].browse(self.partner_id.id)
+        employees = company.child_ids
+        if company and employees:
```

Comment:
No need to check the company and employees both. because ` employees = company.child_ids`  employees gets from company only.